fuel-vm/src/interpreter/executors/main.rs

```diff
@@ -257,6 +281,7 @@ where
                 let tx = kind.tx().clone();
                 let my_params = params.clone();
                 let mut memory = pool.get_new().await;
+                let my_storage = storage.clone();
```

Comment:
jut checking, in practice this is going to result in a shallow copy (e.g. the actual storage is hidden behind an Arc)?

Reply:
Removed